src/main/resources/create_tables.sql

```diff
@@ -0,0 +1,36 @@
+create type roles as enum ('jobseeker', 'recruiter')
+
+CREATE TABLE users (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    first_name VARCHAR(50) NOT NULL,
+    last_name VARCHAR(50) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    gender VARCHAR(7) NOT NULL,
+    email VARCHAR(100),
+    status roles,
+    UNIQUE(email)
+);
+
+CREATE TABLE vacancies (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    vac_name VARCHAR(50),
+    description TEXT,
+    salary DECIMAL(10, 2),
+);
+
+create table company (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL
+);
+
+create table response (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    user_id BIGINT REFERENCES users(id),
+    vacancy_id BIGINT REFERENCES vacancies(id)
+);
+
+create table search (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    user_id BIGINT REFERENCES users(id),
+    vacancy_id BIGINT REFERENCES vacancies(id)
+);
```

Comment:
```suggestion
);

```
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

Reply:
Everything is ready for the next review

Reply:
Did you understand what this comment is about? Did you check the article?

Reply:
Yes, fixed